src/rdbms/rdbms_queries.erl

```diff
@@ -186,13 +208,21 @@ mssql_on_conflict(Updates) ->
 
 update_field_expression(Update) ->
     case is_field(Update) of
+        {true, Expr} -> Expr;
         true -> [Update, " = ?"];
         false -> Update
     end.
 
+is_field({_, FieldExpr}) ->
+    binary:match(FieldExpr, <<"=">>) =/= nomatch andalso {true, FieldExpr};
```

Comment:
Would it be possible to avoid `andalso` here for better readability? It is also a bit strange that `is_field` and `get_field` return very similar results, which makes it difficult to guess their purpose.

Reply:
Specifically, all changes I made to the `rdbms/` files are the dirty ones where I need input on how to improve them. Let me try to fix these comments on these files 😄